plugins/janus_streaming.c

```diff
@@ -5762,6 +5953,40 @@ static void *janus_streaming_relay_thread(void *data) {
 					packet.data = NULL;
 					g_free(text);
 					continue;
+				} else if(audio_rtcp_fd != -1 && fds[i].fd == audio_rtcp_fd) {
+					addrlen = sizeof(remote);
+					bytes = recvfrom(audio_rtcp_fd, buffer, 1500, 0, (struct sockaddr*)&remote, &addrlen);
+					if(bytes < 0) {
+						/* Failed to read? */
+						continue;
+					}
+					JANUS_LOG(LOG_VERB, "Outgoing Audio RTCP SSRC %u\n",
```

Comment:
These new log lines should be prefixed by the `[%s]` mountpoint names as the other ones.